src/drawline.c

```diff
@@ -1223,6 +1223,9 @@ win_line(
 #ifdef FEAT_DIFF
     int		change_start = MAXCOL;	// first col of changed area
     int		change_end = -1;	// last col of changed area
+    diffline_T	line_changes;
+    CLEAR_FIELD(line_changes);
```

Comment:
That looks a bit strange between all the declarations, should this be moved to (above line 1481) instead?

Reply:
DONE